<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registr</class>
 <widget class="QDialog" name="Registr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="Telefonlabel">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Телефон:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Parolabel_2">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Повторите пароль:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Loginlabel">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Parolabel">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Pochtalabel">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Почта:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Polabel">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Пол:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ParoEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="loginEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="ParoEdit_2"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="TelefonEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="PochtaEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QRadioButton" name="MButton">
       <property name="text">
        <string>М</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="JButton">
       <property name="text">
        <string>Ж</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="AdminBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arimo</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Зарегистрироваться как администратор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ZaregButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>145</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
